src/app.component.js

```diff
@@ -154,7 +155,7 @@ class AppComponent extends React.Component {
                         currentSection={this.state.category}
                         showSearchField
                         searchFieldLabel={this.props.d2.i18n.getTranslation('search_settings')}
-                        ref={setSidebar}
+                        ref={setSidebar} // eslint-disable-line react/jsx-no-bind
```

Comment:
I would just move setSidebar out to a method, so we don't need this. Also cleaner then having a function declared in the render-method.